src/components/ProjectPreview.tsx

```diff
@@ -0,0 +1,29 @@
+import { useTheme } from 'styled-components'
+import { PreviewText, PreviewWrapper, PreviewTitle } from './styles/ProjectPreview.styled'
+
+type Props = {
+  title: string
+  text: string
+  hasAnimation?: boolean
+  onClick?: () => void
```

Comment:
consider using `React.MouseEventHandler<HTMLDivElement>` instead of `()=>void`

Reply:
acb0a0e9ab69f76346e7d80a15addbffef9a7514